### DISCUSSION TITLE
Website is error

### DISCUSSION THREAD

**Discussion by DamonGG** [mermaid-js/mermaid#5714] [General]
Website is error:
<img width="954" alt="image" src="https://github.com/user-attachments/assets/6d6edb29-aa2c-4942-88b9-8d1d17e9e13d">

**Comment by sidharthv96** [comment]
<img width="1320" alt="image" src="https://github.com/user-attachments/assets/6b5d0ebb-7798-48b2-94c1-71119b4d041d">

Working now
